### PR TITLE
[CARBONDATA-1326][WIP]: Normal/Low priority findbug fixes

### PR DESCRIPTION
**Pull request by kushalsaha**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[CARBONDATA-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
 - [ ] Testing done
 
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - What manual testing you have done?
        - Any additional information to help reviewers in testing this change.
         
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
                 
---

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: processing/src/main/java/org/apache/carbondata/processing/sortandgroupby/sortdata/SortParameters.java [hunk 1]
```diff
@@ -34,6 +34,8 @@
 
 public class SortParameters implements Serializable {
 
+  private static final long serialVersionUID = 0L;
```
Comment (jackylk) [inline processing/src/main/java/org/apache/carbondata/processing/sortandgroupby/sortdata/SortParameters.java]: serialVersionUID should not be 0L

File: processing/src/main/java/org/apache/carbondata/processing/store/CarbonFactDataHandlerColumnar.java [hunk 2]
```diff
@@ -21,13 +21,7 @@
...
 import java.util.ArrayList;
 import java.util.List;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-import java.util.concurrent.Semaphore;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```
Comment (jackylk) [inline processing/src/main/java/org/apache/carbondata/processing/store/CarbonFactDataHandlerColumnar.java]: do not use *

File: processing/src/main/java/org/apache/carbondata/processing/store/writer/CarbonDataWriterVo.java [hunk 3]
```diff
@@ -155,14 +155,14 @@ public void setIsComplexType(boolean[] isComplexType) {
...
+    return noDictionaryCount;
   }
 
   /**
-   * @param noDictionaryCount the noDictionaryCount to set
+   * @param no_DictionaryCount the noDictionaryCount to set
    */
-  public void setNoDictionaryCount(int noDictionaryCount) {
-    NoDictionaryCount = noDictionaryCount;
+  public void setNoDictionaryCount(int no_DictionaryCount) {
```
Comment (jackylk) [inline processing/src/main/java/org/apache/carbondata/processing/store/writer/CarbonDataWriterVo.java]: No need to change this

**Pull request closed without merging**